### ISSUE TITLE
Refactor information architecture

### ISSUE THREAD

**Issue by cmc333333** [eregs/eregs.github.io#62]
We've been sort of shoving everything in the left-hand navigation rather than thinking about the site's structure. Currently, we have:

* Overview and live instances (which is basically just a link farm to other pages)
* Features
* Code and documentation
* Theming an instance
* History, present, and future
* Introduction to regulations
* Guidelines (how to write regulations)
* Contact and Governance

I propose the following instead:

* Overview (ideally revamped content to be more of a summary)
* History and Trajectory (includes scope section from current history page)
    * Projects and Phases (see #61, includes parts of the "Features" content)
    * In the News (current, on the History page)
    * Presentations (split this from the news part)
* Guides
    * Introduction to regulations
    * Creating an eRegs instance (revised content, collected from ATF docs, /story, Theming an instance, etc.)
    * Architecture (revised content, collected from /technology, read-the-docs, etc.)
    * Writing regulations guide
* Contact and Governance (now to include open source info)

**Comment by nicoleslaw** [comment]
After reading through everything, I really like your suggestion @cmc333333! A couple of other notes:

* Some of the writing could be shorter and more direct. For example the Overview page has headings for things in the nav, but the copy under the headings doesn't add much detail. I don’t mind that we’re pointing people to stuff, but the actual implementation of the idea slows me down as a reader. (Happy to fix that.)

* Instead of `History and Trajectory`, what do you think of flipping the order since there are several success stories now? Ex: `Roadmap and history` or maybe leave it but make it plainer: `Background and next steps`?

* We seem to be defining what eRegs is in several places (Overview, Guidelines, Contact). I think we can do that once in the Overview to save space and duplication. Similar issue with mentioning all the repos.

Does it make sense for you to take a first shot based on your outline, and then for me to do a copyediting pass? Let me know what you prefer!

**Comment by nicoleslaw** [comment]
Having trouble "seeing" all of the content in Markdown at once, so I am reshaping it here: https://docs.google.com/document/d/1xhPOYMhDRqK-6qx3oqDiET-OCwmBU9VM3_Gf8R_kfZo/edit# — and working on the `docs-ia` branch.

**Comment by nicoleslaw** [comment]
👋  @cmc333333 @theresaanna I took a first shot at this and flagged a couple of questions inline: https://docs.google.com/document/d/1xhPOYMhDRqK-6qx3oqDiET-OCwmBU9VM3_Gf8R_kfZo/edit#

Rough outline:

* About the platform
* Introduction to regulations
* Code and repositories
* Creating an eRegulations instance
* Writing rules and regulations
* Contributing and open source
* Contact

I tried to focus on making things flow in a more logical order and cutting duplicatory information. That said, I am concerned about tucking pages or topics under others too frequently because I am not sure if the site design accommodates subsections (for `h2`s) in the navigation. If it doesn't, I can make some small adjustments to make the subpages easier to find.

Anyway, let me know if you want to chat about this. I can set up time for us!
